packages/react-charting/src/components/AreaChart/AreaChartRTL.test.tsx

```diff
@@ -337,6 +339,16 @@ describe('Area chart rendering', () => {
       !(isTimezoneSet(Timezone.UTC) && isTestEnv()),
     );
   });
+
+  testWithoutWait(
+    'Should render the Area chart with secondary Y axis',
+    AreaChart,
+    { data: chartData, secondaryYScaleOptions: secondaryYScalePoints },
+    container => {
+      // Assert
+      expect(getById(container, /yAxisGElementSecondarychart_/i)).toBeDefined();
```

Comment:
make the test to validate the generated snapshot also.

Reply:
Added an assert for snapshot testing.